### ISSUE TITLE
Bytecode disassembler tools

### ISSUE THREAD

**Issue by quiet-node** [hashgraph/hedera-mirror-node-explorer#764]
### Problem

A new enhancement to support bytecode disassembling and bytecode decompiling is proposed 

### Solution

- Create a new route for the tools in question
- Add bytecode disassembler tool
- Add bytecode decompiler tool
- Add new components to let users interact with the tools

```[tasklist]
### Tasks
- [ ] https://github.com/hashgraph/hedera-mirror-node-explorer/issues/817
```

**Comment by SimiHunjan** [comment]
Split this into two issues. One for disassembler and one for decompiler

**Comment by SimiHunjan** [comment]
Need to review what was done and identify any unknowns

**Comment by Nana-EC** [comment]
I believe we completed the disassembler part but got blocked on the decompiler.
I'm looking for the issue capturing the progress and blocker there

**Comment by SimiHunjan** [comment]
Ah I didn't get around to splitting the issue. Will do now.

**Comment by SimiHunjan** [comment]
Created issue here https://github.com/hashgraph/hedera-mirror-node-explorer/issues/857 @Nana-EC 

Will close this one as complete for the bytecode disassembler
